### PR TITLE
Adds youtube embed partial and embeds getting started video in Get Started page

### PR DESCRIPTION
**Pull request by tomglenn**

### CODE REVIEW

**Review by sympatheticmoose — APPROVED** [review]

**Review by jackw — DISMISSED** [review]
LGTM! 🚀

**Comment by tomglenn** [comment]
Thanks @jackw , I've updated this to use a React component.

**Review by leventebalogh — APPROVED** [review]
Left one comment, but apart from that LGTM 👍

**Review by jackw — APPROVED** [review]
LGTM. 👍 

Agree with Levi's suggestion to make props required.

**Comment by tomglenn** [comment]
@leventebalogh @jackw I've updated the `YouTubeEmbedProps` type as suggested, but I've left the conditional check inside the component since there doesn't appear to be any compile time enforcement of prop types when used within MDX files. I assume this is a quirk of Docusaurus and the way it compiles `mdx` files.

File: docusaurus/website/src/components/YouTubeEmbed/YouTubeEmbed.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Define the type to control the props that can be passed to the YouTubeEmbed component.
+type YouTubeEmbedProps = {
+  // The YouTube video Id to embed.
+  videoId?: string;
+  // The title of the YouTube video for accessibility.
+  title?: string;
+};
```
[the remark this answers is not in the document]
Comment (leventebalogh) [reply docusaurus/website/src/components/YouTubeEmbed/YouTubeEmbed.tsx:9]: Hmm, interesting, I didn't know that! Well, then it's probably not that crucial to change (however I see that you have changed it already :)). If this behaviour would be "fixed" at any point in the future then at least now we are safe :)

**Review by leventebalogh — APPROVED** [review]

**Pull request merged**